content/blog/2023/2023-08-24-fairness-reweighing-dataset.md

```diff
@@ -0,0 +1,41 @@
++++
+author = "Žan Mervič"
+date = "2023-08-24"
+draft = false
+title = "Orange Fairness - Reweighing a Dataset"
+type = "blog"
+thumbImage = "/blog_img/2023/2023-08-24-fairness-reweighing-dataset-thumb.png"
+frontPageImage = "/blog_img/2023/2023-08-24-fairness-reweighing-dataset-thumb.png"
+blog = ["fairness", "reweighing"]
+shortExcerpt = "Introducing the Reweighing widget in Orange, a solution for dataset bias mitigation."
+longExcerpt = "Building on our exploration of the Orange fairness addon, this blog delves into the Reweighing widget. By adjusting weights for dataset instances, the widget addresses bias, focusing on underrepresented groups. Using the Compas dataset as an example, we demonstrate how bias decreases post-reweighting, presenting visual insights into the distribution of adjusted weights and their impact on dataset fairness."
+
++++
+
+In the [previous blog post]({{< ref "/blog/2023/2023-08-23-fairness-dataset-bias.md" >}}), we introduced the Orange fairness addon along with the Dataset Bias and As Fairness widgets. We also demonstrated how to use them to detect bias in a dataset and visualized the results for a better understanding. In this blog, we will introduce the Reweighing widget, which we can use to mitigate bias in a dataset, resulting in fairer machine learning models learning from it.
```

Comment:
Don't link to .md. This link should probably point to "/blog/2023/2023-08-23-fairness-dataset-bias/"). Then the build would not fail.